disk/disk_unix.go

```diff
@@ -2,11 +2,11 @@
 
 package disk
 
-import "syscall"
+import "golang.org/x/sys/unix"
 
 func Usage(path string) (*UsageStat, error) {
-	stat := syscall.Statfs_t{}
-	err := syscall.Statfs(path, &stat)
+	stat := unix.Statfs_t{}
```

Comment:
Sorry for my laziness since #30, we should move to `x/sys/unix`. I agree. However, I think this migration is not a part of this PR. So could you remove from this PR? Or, if  `x/sys/unix` is required to work on illumos, I will make a new PR to migrate to `x/sys/unix` (but it takes a little time).